packages/okta-react/test/e2e/harness/src/SessionTokenLogin.js

```diff
@@ -62,7 +63,6 @@ export default withAuth(class SessionTokenLogin extends Component {
 
   render() {
     if (this.state.sessionToken) {
-      this.props.auth.redirect({sessionToken: this.state.sessionToken});
```

Comment:
ditto

Reply:
same explanation. Unlike our courage library, React components will render whenever state / props are changed. when using our `withAuth` higher order component it will trigger render for us.  We want to redirect once and ONLY once.